docs/tools/firmware.sh

```diff
@@ -18,36 +18,178 @@ while getopts "vhx" option; do
 	esac
 done
 
+prerequisite_packages () {
+cat <<EOF
+
+Prerequisites needed to continue:
+
+1. Python 3
+2. ${additional_requirement}
+
+Python 3 is a part of Xcode command line tools. You can also use Homebrew, MacPorts etc.
+Continue when you have the prerequisites installed.
+
+Continue? (Y/n)
+EOF
+}
+
+create_deb () {
```

Comment:
Indent this whole function

Reply:
As in add it to the macOS only part?

Reply:
I mean just press tab in front of the contents of the create_deb function so its easy to see where it ends, but its not really that important